src/check/runner/configuration/GlobalParameters.ts

```diff
@@ -4,11 +4,21 @@ import { Parameters } from './Parameters';
 /** @internal */
 const globalParametersSymbol = Symbol.for('fast-check/GlobalParameters');
 
+export type GlobalPropertyHookFunction = () => void;
+export type GlobalAsyncPropertyHookFunction = (() => Promise<unknown>) | (() => void);
```

Comment:
As any exported type that are used by types or functions exposed to the end users they will need to be marked as public, documented and exported in fast-check-default.ts file.